---
name: Feature request
about: Suggest an idea or improvement for this ESLint configuration
title: "[Feature] "
labels: "enhancement"
assignees: ""

---

**Is your feature request related to a problem? Please describe.**  
A clear and concise description of what the pain point is. For example, "I get false positives when ..." or "The current rules don't catch ...".

---
### Describe the solution you'd like
A clear and concise description of what you want to happen.
If you are requesting new rules or rule options, please include:

- Rule name(s) (if existing ESLint/core or plugin rule)
- Proposed rule level/config (`error`, `warn`, etc.)
- Examples of **incorrect** code that should be flagged
- Examples of **correct** code that should be allowed

---
### Describe alternatives you've considered
Explain other configurations, tools, or approaches you've tried.

---
### Additional context
Add any other context or screenshots about the feature request here.

---
### Are you willing to submit a pull request?
- [ ] Yes, I can create a PR implementing this feature.
- [ ] No
